### DISCUSSION TITLE
prepend to list

### DISCUSSION THREAD

**Discussion by sina-devel** [vlang/v#11668] [Ideas]
```
>>> mut arr := [3, 4]
>>> [1, 2] >> arr
[1, 2, 3, 4]
```

**Comment by JalonSolov** [comment]
I would expect >> to be the opposite of <<... so if << adds something to the end of an array, >> should remove it, not add something to the top.

Of course, I see the argument for this as well... if both add, then one adds to top while other adds to bottom... it just looks odd.

And there are no matching remove symbols, using both of these to mean add.
